src/enums/entity-types.ts

```diff
@@ -11,6 +11,7 @@ export enum EntityTypes {
   CCC = CorpTypeCd.CCC_CONTINUE_IN,
   CP = CorpTypeCd.COOP,
   CR = CorpTypeCd.CORPORATION,
+  CS = CorpTypeCd.CONT_IN_SOCIETY,
```

Comment:
... Even though we have that other feature flag to redirect users straight to Society Online if they choose a society-anything NR.